rector.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Set\ValueObject\LevelSetList;
+use Rector\Symfony\Set\SymfonySetList;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->sets([
+        LevelSetList::UP_TO_PHP_81,
```

Comment:
Depending on [this comment](https://github.com/BedrockStreaming/RateLimitBundle/pull/9#discussion_r934296618), it should probably be adapted

Reply:
done